app/src/main/java/com/babylon/wallet/android/presentation/transaction/TransactionReviewViewModel.kt

```diff
@@ -357,7 +357,9 @@ class TransactionReviewViewModel @Inject constructor(
             get() = feePayers?.selectedAccountAddress == null
 
         val isSelectedFeePayerInvolvedInTransaction: Boolean
-            get() = request?.transactionManifestData?.feePayerCandidates()?.contains(feePayers?.selectedAccountAddress) ?: false
+            get() = runCatching {
```

Comment:
This fixes the crash when the `TransactionManifest.init` from `TransactionManifestData` throws.